src/PhpSpreadsheet/Calculation/Engineering/BesselI.php

```diff
@@ -111,9 +111,16 @@ private static function besselI1(float $x): float
         return ($x < 0.0) ? -$ans : $ans;
     }
 
+    /**
+     * Sop to Scrutinizer.
+     *
+     * @var float
+     */
+    private static $zeroPointZero = 0.0;
+
     private static function besselI2(float $x, int $ord): float
     {
-        if ($x === 0.0) {
+        if ($x === self::$zeroPointZero) {
             return 0.0;
```

Comment:
Does it work with a constant as well as with a static variable?

Reply:
I didn't try with a constant. I'll set up a test later today.

Reply:
See PR #3123, which I will delete in a day or two. Using a constant did not eliminate Scrutinizer's objection.

Reply:
Type-hinting constant also didn't help. Sticking with static.